client/locales/en.yml

```diff
@@ -149,16 +149,30 @@ pages:
     title: Favorites
   upload:
     title: Upload
+    dropzone:
+      title: Everything starts here
+      description: Drag and drop files in this area to upload them.
+      drop_here: Drop
+    status:
+      pending: Pending
+      uploading: Uploading
+      completed: Completed
+      error: Failed
+    success:
+      title: Upload completed
+      detail: "{count} file has been uploaded | {count} files have been uploaded"
+    error:
+      title: Upload failed
+      detail: "{count} file could not be uploaded | {count} files could not be uploaded"
+    partial:
+      title: Upload completed with errors
+      detail: "{count} file out of {total} has been uploaded | {count} files out of {total} have been uploaded"
     drag_n_drop: Drag and drop files here to upload.
```

Comment:
missing key in fr + may be unused since redundant with pages.upload.dropzone.description